src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/LayerDescriptor.java

```diff
@@ -59,6 +65,8 @@ public String getUrlFieldName() {
         return urlField;
     }
 
+    public String getVariableName() { return variable; }
```

Comment:
This has always bugged me and caused anissue with Dan over it. `netCDFVariable` rocks!